### ISSUE TITLE
False positive on parentheses used for line continuation

### ISSUE THREAD

**Issue by w0rp** [flake8-commas/flake8-commas#19]
PEP8 recommends using parentheses, and not a `\` character for implicit line continuation. I use this style often for writing Django code.

``` python
queryset = (
    MyModel.objects
    .filter(...)
    .select_related(...)
)
```

When I use lines like the above, this plugin incorrectly believes that a trailing comma is missing from the above expression.

**Comment by graingert** [comment]
@w0rp fixed in 0.2.0

**Comment by graingert** [comment]
of course if you use a function:

```python
queryset(
    MyModel.objects
    .filter(...)
    .select_related(...),
)
```

the comma is mandatory even though it looks odd!

**Comment by w0rp** [comment]
Interesting. I shall have to try this plugin again.
